src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -200,16 +213,7 @@
        (p/catch (fn [error]
                   (js/alert error))))))
   (delete-file! [_this repo dir path {:keys [ok-handler error-handler]}]
-    (let [path (cond
-                 (= (mobile-util/platform) "ios")
-                 (js/encodeURI (js/decodeURI path))
-
-                 (string/starts-with? path (config/get-repo-dir repo))
```

Comment:
`(config/get-repo-dir repo)` returns `nil` when adding a fresh new graph, which leads to an illegal file-path for creating config/theme/metadata files.